charts/fluent-bit/values.yaml

```diff
@@ -141,6 +141,39 @@ resources: {}
 #     cpu: 100m
 #     memory: 128Mi
 
+## only available if kind is Deployment
+autoscaling:
+  enabled: false
+#  minReplicas: 1
+#  maxReplicas: 100
+#  targetCPUUtilizationPercentage: 80
+#  targetMemoryUtilizationPercentage: 80
```

Comment:
These values are named in the template so should be uncommented with either "good/tested" defaults or empty values and defaults in comments.

Reply:
I don't think the defaults you've added are necessarily correct. Having 100 max pods seems very high and wouldn't be good for a user to accidentally use, I'd suggest 3 maybe? RE the utilization I'd suggest leaving them empty as you're not necessarily going to want them set.

Reply:
max changed to 3, commented out memory utilization, but leave cpu utilization as is, it must scale according something